File: example/basicjava/3-multi-module/bar/test/src/BarTests.java
package bar;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class BarTests {

    @Test
    public void simple() {
        String result = Bar.generateHtml("hello");
        assertEquals("<h1>hello</h1>", result);
    }

    @Test
    public void escaping() {
        String result = Bar.generateHtml("<hello>");
        assertEquals("<h1>&lt;hello&gt;</h1>", result);
    }
}

File: example/kotlinlib/basic/3-multi-module/bar/test/src/bar/BarTests.kt
package bar

import bar.generateHtml
import org.junit.Assert.assertEquals
import org.junit.Test

class BarTests {

    @Test
    fun simple() {
        val result = generateHtml("hello")
        assertEquals("<h1>hello</h1>", result)
    }

    @Test
    fun escaping() {
        val result = generateHtml("<hello>")
        assertEquals("<h1>&lt;hello&gt;</h1>", result)
    }
}
